yb-voyager/cmd/export.go

```diff
@@ -103,9 +108,43 @@ func registerCommonExportFlags(cmd *cobra.Command) {
 
 	cmd.Flags().StringVar(&source.SSLCRL, "source-ssl-crl", "",
 		"source SSL Root Certificate Revocation List (CRL)")
+}
 
-	cmd.Flags().BoolVar(&startClean, "start-clean", false,
-		"cleans up the project directory for schema or data files depending on the export command")
+func registerTargetDBAsSourceConnFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&source.Host, "target-db-host", "127.0.0.1",
+		"host on which the YugabyteDB server is running")
+
+	cmd.Flags().IntVar(&source.Port, "target-db-port", -1,
+		"port on which the YugabyteDB YSQL API is running")
```

Comment:
I hope this works when user doesn't provide `--target-db-port`.

Reply:
Ah, yes. `setSourceDefaultPort` takes care of YB as well.